src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int Figure_Count = 6;
```

Comment:
This variable should be a class constant.
Please follow the rules of constants naming:
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)